druid/src/widget/sized_box.rs

```diff
@@ -76,6 +76,33 @@ impl<T> SizedBox<T> {
         self
     }
 
+    fn child_constraints(&self, bc: &BoxConstraints) -> BoxConstraints {
+        // if we don't have a width/height, we don't change that axis.
+        // if we have an infinite width/height, we tighten that axis
```

Comment:
I'm not following how this behaves differently for infinite and non-infinite? Just that infinite will always be the max?

Reply:
fair point, that was confusing, I reworded this comment.